src/vlei-verification.ts

```diff
@@ -65,12 +65,14 @@ export class VleiVerification {
     credCesr: string,
     expected_status_code: number
   ) {
-    const verifierResponse = await this.verifierClient.login(said, credCesr);
-    assert.equal(verifierResponse.code, expected_status_code);
+    await this.verifierClient.login(said, credCesr).then((res) => {
```

Comment:
i see the comment, i believe the assertion is making sure the login called executed with a non-error-code

Reply:
@jrayback let me know if you agree and if you will remove the comment or adjust the code.